src/App.java

```diff
@@ -1,5 +1,203 @@
+import java.util.Scanner;
+
 public class App {
+    public Random myRandom;
+    // creates a connection to the other two classes, Random.java and
+    // InputOutput.java
+    private InputOutput io = new InputOutput();
+    public int numStudents = 0;
+    // numStudents is how many students are the
+    public int studentsPerTable = 0;
+    // studentsPerTable is later established when the user sets it as whatever they
+    // would like
+    private int option = 1;
+    // this is used for the user to select what option they would like.
+    private String filename = "";
+    private String option2 = "";
+    // This will be used to name the file the user would like to create
+    private Boolean fileError = true;
+    private Boolean initialFileError = true;
+    private String names = "";
+
+    /**
+     * Where the program begins!
+     */
+    public void start() {
+       
+        openingMenu();
+    }
+
+    // This is utilzied to read the file the user has set up. It is able to count

```

Comment:
You should use documentation comments that look like 
```
/**
    comment
/*
```
as the comments before methods